DSL.Resql/get_task.sql

```diff
@@ -0,0 +1 @@
+select *, sentences_annotations::text AS predictions from Corpora_Tasks where corpora_id = :corpora_id and id = :id ORDER BY created_at DESC limit 1;
```

Comment:
Avoid `select *` in any case and be explicit on what you are requesting.

Reply:
Done